core/src/ldml/ldml_processor.cpp

```diff
@@ -316,13 +316,26 @@ size_t ldml_processor::process_output(km_core_state *state, const std::u32string
   // drop last 'matchedContext':
   ctxtstr.resize(ctxtstr.length() - matchedContext);
   ctxtstr.append(outputString); // TODO-LDML: should be able to do a normalization-safe append here.
-  ldml::marker_map markers;
-  assert(ldml::normalize_nfd_markers(ctxtstr, markers)); // TODO-LDML: Need marker-safe normalize here.
+  {
+    const auto normalize_ok = ldml::normalize_nfd_markers(ctxtstr);
+    assert(normalize_ok);
+    if(!normalize_ok) {
+      DebugLog("ldml_processor::process_output: failed ldml::normalize_nfd_markers(ctxtstr)");
+    }
+  }
```

Comment:
Do we need to abort processing in release builds?

Reply:
If it fails the string is just unchanged and so the match may not work properly or wrong output. But there's no inherent danger in proceeding.

Is it worth trying to pop up an error code to the engine? I thought the answer before was, no.

Reply:
No danger is what I was looking for. There's no way we can pop an error, so 'bad output' is the best error we can do. (And no, don't even think about emitting an error as key events "YOUR KEYBOARD HAS GONE WRONG" emitted every time you press a key?!)